example.lua

```diff
@@ -43,11 +49,6 @@ end
 -- upon successful parsing cliargs will delete itslef to free resources
 -- for k,item in pairs(args) do print(k .. " => " .. tostring(item)) end
 
--- checking for flags: is -v or --version set?
-if args["v"] then
-  return print("cli_example.lua: version 1.2.1")
```

Comment:
i would keep this in the example though (likely with a different parameter) as your change doesn't deprecate this use-case